include/ppx/application.h

```diff
@@ -490,9 +490,10 @@ class Application
 
 #if defined(PPX_BUILD_XR)
     XrComponent mXrComponent;
-    uint32_t    mDebugCaptureSwapchainIndex = 0;
-    uint32_t    mUISwapchainIndex           = 0;
-    uint32_t    mStereoscopicSwapchainIndex = 0;
+    uint32_t    mDebugCaptureSwapchainIndex  = 0;
+    uint32_t    mUISwapchainIndex            = 0;
+    uint32_t    mStereoscopicSwapchainIndex  = 0;
+    int         mTogglePassthroughFrameCount = 0;
```

Comment:
Why is this a signed int, since you check for `mTogglePassthroughFrameCount > 0`? Looks like is could be an uint?

Reply:
I removed the field.